### PR TITLE
Updates and fixes to symetrie hexapod

### PR DESCRIPTION
**Pull request by GDYendell**
These are changes required to get the symetrie hexapod on I07 working. The key things are:

- Update PVs with PITCH, YAW, ROLL to use RX, RY, RZ to make it more generic
- Fix bit mask statuses
- Fix RESETDEMANDS demand sync logic
- Sync RBV to demand when a custom symetrie offset changes - to match motor record behaviour
- Two new EDM screens and updates to the existing summary screen

### CODE REVIEW

File: etc/builder.py [hunk 1]
```diff
@@ -747,19 +747,25 @@ class SymetrieHexapodTemplate(AutoSubstitution):
...
     def Initialise(self):
         print('pmacNoCsVelocity("%(PORT)s")' % self.__dict__)
 
     ArgInfo = makeArgInfo(
         __init__,
+        name = Simple("Object name", str),
         P=Simple("Device prefix", str),
         PORT=Ident("GeoBrick object", GeoBrick),
         COORD=Simple("Coordinate system number", int),
+        RX=Simple("Name of RX axis (axis 4)", str),
```
Comment (EmilioPeJu) [inline etc/builder.py]: Maybe the description for RX, RY and RZ should mention that it's used in GUI only? someone not familiar with this change might wrongly think that this affects the PV names

**Review by EmilioPeJu — APPROVED** [review]
Looks good to me

**Comment by EmilioPeJu** [comment]
It looks like the CI is broken because it depends on some broken repository for installing packages, I suppose we can ignore that for now, as we are moving from Travis to Github actions

**Comment by gilesknap** [comment]
Thanks Gary, I'm not worried about the CI - as Emilio says its broken anyway. You did not change anything that would affect the unit tests anyway.

**Pull request merged**